OpenSim/Simulation/OpenSense/InverseKinematicsStudy.cpp

```diff
@@ -195,11 +198,11 @@ runInverseKinematicsWithOrientationsFromFile(Model& model,
     std::string outputFile = get_results_directory() + "/" + outName;
 
     // Convert to degrees to compare with marker-based IK
-    // but ignore translational coordinates
+    // but only for rotational coordinates
     for (size_t i = 0; i < report.getNumColumns(); ++i) {
-        auto it = find( transCoordNames.begin(), transCoordNames.end(),
+        auto it = find(rotCoordNames.begin(), rotCoordNames.end(),
                         report.getColumnLabel(i) );
-        if (it == transCoordNames.end()) { // not found = not translational
+        if (it != rotCoordNames.end()) { // found a rotational coordinate
             auto repVec = report.updDependentColumnAtIndex(i);
             repVec *= SimTK::Real(SimTK_RTD);
         }
```

Comment:
Any reason not to use the method convertRadianToDegrees e to have these checks and conversions done in one pace?

Reply:
I didn't want to convert to a `Storage`, but I see that Shrinidhi added a `TimeSeriesTable` version. I'll try that.

Reply:
I remember now,  conversion fails if the "inDegrees" key is not specified. I've fixed that now too.